docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,50 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 26 April 2023
+
+Frontend version: 1.0.0-20230413.1713 | Backend version: 1.61.1-230420.1127
```

Comment:
I see these versions in our DEV environment.

frontend: 1.0.0-20230424.1714
backend: 1.61.3-230425.0821

Reply:
@yqyeoh any updates? on TSD, thats the numbers (on release notes)

Reply:
ah sorry, follow jerome's one, i've updated the TSD